sklego/linear_model.py

```diff
@@ -606,9 +623,9 @@ def fit(self, X, y, sample_weight=None):
             loss,
             x0=np.zeros(self.n_features_in_ + d),
             bounds=bounds,
-            method="SLSQP",
+            method=self.method,
             jac=grad_loss,
-            tol=1e-20,
+            tol=1e-6,
```

Comment:
Is it necessary that the default here changed?

Reply:
I guess it's a leftover, for SLSQP default tol (ftol) is 1e-6, for tnc it's 0 and for lbfgsb it's 2.220446049250313e-09, so I change it to maximum among the three. Should I revert it to 1e-20?

Reply:
Yea that'd be good as it's technically an unnecessary breaking change now.